packages/at_lookup/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 3.0.48
+- feat: consume EnrollVerbBuilder in AtLookup.executeVerb()
+- chore: upgrade at_commons to v4.1.1 and at_Utils to v3.0.18
```

Comment:
change the case at_Utils --> at_utils

Reply:
change made in [a293b1f](https://github.com/atsign-foundation/at_libraries/pull/657/commits/a293b1f2bb64475a43ed1bce3598f92ce828420c)